.python-version

```diff
@@ -0,0 +1 @@
+3.9.0
```

Comment:
This file should not be in source control. I have mine configured to use a virtualenv named `metecho`

Reply:
This should only impact devs using pyenv, but I'll remove it.